### DISCUSSION TITLE
GitHub Campus Experts

### DISCUSSION THREAD

**Discussion by schmelto** [EddieHubCommunity/support#832] [General] [answered]
Discussion about GitHub Campus Expert.

@eddiejaoude maybe you can post here some more information how #eddiehub members can sign up for the program.

**Comment by varghesejose2020** [comment]
GitHub Campus Expert is only for students @schmelto

**Comment by aniketsingh98571** [comment]
@schmelto Yeah, @varghesejose2020  is right Github Campus Experts has nothing to do with EddieHub, GCE is run by Github, not EddieHub.

**Comment by Prathamesh-Shanbhag** [comment]
I guess there are a lot of students? Also I think to be asked for guidance is not wrong ,even though EddieHub might not be related to it!

**Comment by eddiejaoude** [comment, marked as answer]
Activity community members of EddieHub, have a special link they can sign up for to become a GitHub Campus Expert, provided by GitHub Education
